### PR TITLE
JOSE Paper

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#9

### CODE REVIEW

File: paper/paper.bib [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@book{Lawler:1995,
```
Comment (sepandhaghighi) [inline paper/paper.bib]: @sadrasabouri I'm not sure about this citation format.
Where did you get this?

File: paper/paper.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+date: 15 August 2021
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Pyrandwalk is an educational tool for simulating random walks, calculating the probability of given state sequences, etc. Random walk is a representation of the discrete-time, discrete-value Markov chain model used in stochastic processes.
+
+This paper format is inherited from Pyrgg.[@Haghighi2017]
```
Comment (sepandhaghighi) [inline paper/paper.md]: The first idea of this paper is inherited from Pyrgg

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

**Comment by sadrasabouri** [comment]
[PyRandWalk.pdf](https://github.com/sadrasabouri/pyrandwalk/files/7032835/nl0q7vzwtwuii429xszr.pdf)

File: paper/paper.bib (hunk 1 above)
Comment (sadrasabouri) [reply paper/paper.bib]: Yes that was not OK
I fetch the right one from Google Scholar.

**Review by sepandhaghighi — APPROVED** [review]
@sadrasabouri LGTM 💯

**Pull request merged**